File: Lista pt3/Exercicio04Arrays.java
public class Exercicio04Arrays {
    public static void main(String[] args) {

        String[][] carros = {
                {"Volkswagen", "Gol", "Polo", "Virtus"},
                {"Chevrolet", "Onix", "Tracker", "Cruze"},
                {"Fiat", "Argo", "Mobi", "Cronos"},
                {"Toyota", "Corolla", "Yaris", "Hilux"}
        };

        System.out.println("MARCA       | MODELO");
        System.out.println("----------------------");

        for (String[] marca : carros) {

            for (int i = 1; i < marca.length; i++) {

                System.out.println(
                        marca[0] + " | " + marca[i]
                );
            }

            System.out.println("----------------------");
        }
    }
}

File: Lista pt3/Exercicio04Arrays.js
let carros = [
    {
        marca: "Toyota",
        modelos: ["Corolla", "Yaris", "Hilux"]
    },
    {
        marca: "Volkswagen",
        modelos: ["Gol", "Polo", "Virtus"]
    },
    {
        marca: "Chevrolet",
        modelos: ["Onix", "Tracker", "Cruze"]
    },
    {
        marca: "Fiat",
        modelos: ["Argo", "Mobi", "Cronos"]
    }
];

let tabela = document.createElement("table");

tabela.border = "1";

for (let i = 0; i < carros.length; i++) {

    for (let j = 0; j < carros[i].modelos.length; j++) {

        let linha = document.createElement("tr");

        let colunaMarca = document.createElement("td");
        let colunaModelo = document.createElement("td");

        colunaMarca.textContent = carros[i].marca;
        colunaModelo.textContent = carros[i].modelos[j];

        linha.appendChild(colunaMarca);
        linha.appendChild(colunaModelo);

        tabela.appendChild(linha);
    }

    let linhaSeparadora = document.createElement("tr");

    let colunaSeparadora = document.createElement("td");

    colunaSeparadora.colSpan = 2;

    let linha = document.createElement("hr");

    colunaSeparadora.appendChild(linha);

    linhaSeparadora.appendChild(colunaSeparadora);

    tabela.appendChild(linhaSeparadora);
}

document.body.appendChild(tabela);